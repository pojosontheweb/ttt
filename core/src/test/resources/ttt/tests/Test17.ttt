<%@ page contentType="text/html" %>
<%!
    int bar;
%>
<div class="foo">
    Salut RÃ©mi
</div>