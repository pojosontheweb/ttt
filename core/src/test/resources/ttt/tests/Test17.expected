package ttt.tests;

public class Test17 extends com.pojosontheweb.ttt.Template {

	private final int bar;

	/**
	 * Creates an instance of this template.
	 *
	 */
	public Test17(int bar) {
		this.bar = bar;
	}

	@Override
	public String getContentType() {
		return "text/html";
	}

	@Override
	public void doRender(com.pojosontheweb.ttt.TttWriter out) throws Exception {
		write(out, "<div class=\"foo\">\n    Salut RÃ©mi\n</div>" );
	}
}
